### PR TITLE
Statistics Graph Editor to Ha Form

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts [hunk 1]
```diff
@@ -78,133 +71,75 @@ export class HuiStatisticsGraphCardEditor
...
-    return this._config!.period || "hour";
-  }
+  private _schema = memoizeOne((localize: LocalizeFunc) => [
+    { name: "title", selector: { text: {} } },
+    {
+      name: "",
+      type: "grid",
+      schema: [
+        {
+          name: "period",
```
Comment (balloob) [inline src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:81]: This is required.

File: src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts [hunk 2]
```diff
@@ -78,133 +71,75 @@ export class HuiStatisticsGraphCardEditor
...
+              options: periods.map((period) => ({
+                value: period,
+                label: localize(
+                  `ui.panel.lovelace.editor.card.statistics-graph.periods.${period}`
+                ),
+              })),
+            },
+          },
+        },
+        { name: "days_to_show", selector: { number: { min: 1, mode: "box" } } },
```
Comment (balloob) [inline src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:93]: required

File: src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts [hunk 3]
```diff
@@ -78,133 +71,75 @@ export class HuiStatisticsGraphCardEditor
...
+        {
+          name: "stat_types",
```
Comment (balloob) [inline src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:95]: required

File: src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts [hunk 4]
```diff
@@ -78,133 +71,75 @@ export class HuiStatisticsGraphCardEditor
...
+          type: "multi_select",
+          options: [
+            ["mean", "Mean"],
+            ["min", "Min"],
+            ["max", "Max"],
+            ["sum", "Sum"],
+          ],
+        },
+        {
+          name: "chart_type",
```
Comment (balloob) [inline src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts:105]: required

**Review by bramkragten — APPROVED** [review]

**Pull request merged**